install/helm/gloo/templates/1-gloo-deployment.yaml

```
@@ -8,8 +8,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app: gloo
-    gloo: gloo
+    {{- include "gloo.labels" . }}
```

```suggestion
    gloo: gloo
    {{- include "gloo.labels" . }}
```
And so for all other occurrences of `gloo: gloo` that has been removed